test/adapter/json/collection_test.rb

```diff
@@ -85,6 +85,72 @@ def test_include_option
 
             assert_equal(expected, actual)
           end
+
+          def test_fields_option
+            serializer = ArraySerializer.new([@first_post, @second_post])
+            adapter = ActiveModel::Serializer::Adapter::Json.new(serializer)
+            actual = adapter.serializable_hash(fields: [:id])
```

Comment:
Looking good 

1. this needs an integration (controller) test
2. Please use `ActiveModel::SerializableResource.new([@first_post, @second_post], fields: [:id])`

Please don't test passing options into `serializable_hash` since only the renderer would do that